src/js/components/Beacon/BeaconCommon/BeaconQueryFormUi.tsx

```diff
@@ -227,7 +227,9 @@ const BeaconQueryFormUi = ({
   return (
     <div className="container margin-auto" style={{ paddingBottom: 8 }}>
       <Card title={t('Search')} className="w-full shadow rounded-xl" styles={CARD_STYLES}>
-        <p style={{ margin: '-8px 0 8px 0', padding: '0', color: 'grey' }}>{t(uiInstructions)}</p>
+        <p className="p-0" style={{ margin: '-8px 0 8px 0', color: 'grey' }}>
```

Comment:
might be better to change the padding of the header of the card or instead of using card title use antd title not in the card header.